corrscope/gui/view_mainwindow.py

```diff
@@ -490,6 +503,8 @@ def retranslateUi(self, MainWindow):
         self.render__bg_colorL.setText(tr("Background"))
         self.render__init_line_colorL.setText(tr("Line Color"))
         self.render__line_widthL.setText(tr("Line Width"))
+        self.render__init_line_outline_colorL.setText(tr("Outline Color"))
+        self.render__line_outline_widthL.setText(tr("Outline Width"))
```

Comment:
This stuff is auto-converted from Qt Designer .ui xml. My preference is to place the labels inline. Not sure the best idea. (also tbh i doubt this UI is translatable at all.)